src/components/rmrk/Gallery/GalleryItem.vue

```diff
@@ -334,7 +334,10 @@ hr.comment-divider {
     margin: 30px auto;
     width: 100%;
 
-    .image {
+    .nft-image img {
+      max-height: 530px;
```

Comment:
I think this will be broken on mobile

Reply:
### Screenshot
#### Mobile View large image
![Screenshot 2021-09-27 at 17-17-55 Decorator Tiles #2 - Dune special](https://user-images.githubusercontent.com/9987732/134937058-1abddeff-85c1-4886-a6d8-201915806e5c.png)
#### Mobile View long image
![Screenshot 2021-09-27 at 17-20-29 Kusamagen #160](https://user-images.githubusercontent.com/9987732/134937516-285759d4-913a-47e2-9171-1bbc2abcd97b.png)